src/components/container/SurveyDataChart/SurveyDataChart.tsx

```diff
@@ -0,0 +1,176 @@
+import React, { useState, useEffect, useRef, useContext } from 'react'
+import { DateRangeContext } from '../../presentational/DateRangeCoordinator/DateRangeCoordinator'
+import { DailyDataProvider, DailyDataQueryResult, checkDailyDataAvailability, queryDailyData } from '../../../helpers/query-daily-data'
+import { add, format, parseISO } from 'date-fns'
+import MyDataHelps, { SurveyAnswer, SurveyAnswersPage } from '@careevolution/mydatahelps-js'
+import getDayKey from '../../../helpers/get-day-key'
+import { WeekStartsOn, getMonthStart, getWeekStart } from '../../../helpers/get-interval-start'
+import DataChart, { AreaChartOptions, BarChartOptions, LineChartOptions } from '../../presentational/DataChart/DataChart'
+
+export interface SurveyAnswerChartLine {
+	label: string;
+	surveyName: string;
+	stepIdentifier: string;
+	resultIdentifier: string;
```

Comment:
Actually maybe just have this take surveyAnswerQueryParameters?  I don't think the abstraction is useful and could get in the way of certain use cases, like if you have pain scores being captured from multiple distinct steps and you want to graph them as the same line.  surveyAnswerQueryParameters should let you pass in multiple result identifiers etc

Reply:
Well, the main driver was that the SurveyAnswerQueryParameters has parameters for time range which we're setting via the `DateRangeContext` - having this parameter be an object with fields that we might *ignore* seemed a bad interface since there's no way to indicate that we'd discard them (or if we didn't discard them, that they'd override the `DateRangeContext`).

As for limiting the other use cases - I'm not sure I understand how you'd graph multiple steps from the same survey on the same line?  The X-axis is time, so multiple responses from the same survey would be at the same x location, which means they can't be on the same line?  They'd have to be parallel lines, and so you'd configure them separately?